wordpress-mobile/WordPress-iOS
# Add Notifications screen for unified prologue carousel

Ref #15056. This PR adds the Notifications screen to the prologue carousel.

| iPhone | iPad |
|---|---|
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-03-29 at 10 37 37](https://user-images.githubusercontent.com/4780/112818083-0e496400-907b-11eb-8ad3-32d1bc1c3d94.png) | ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2021-03-29 at 10 41 30](https://user-images.githubusercontent.com/4780/112818640-a9423e00-907b-11eb-9742-6648d5ade817.png) |

**To test**

* Build, run and log out if necessary
* Verify that the notifications (third) page of the login prologue is consistent with the designs (see issue #15056; see also example screenshot below)

PR submission checklist:
